pnpm-lock.yaml

```diff
@@ -1055,6 +1055,21 @@ importers:
         specifier: ^2.8.0
         version: 2.8.0(solid-js@1.8.7)(vite@5.0.10)
 
+  examples/solid/bug-test:
```

Comment:
the changes to the lock file look weird given that we have no changes in any `package.json` file. Can we revert them please ?

Reply:
Ah yes, I was doing some experiments, didn't realise I updated the lockfile. Will revert tomorrow